pex/pex_builder.py

```diff
@@ -207,17 +208,19 @@ def set_script(self, script):
     """
 
     # check if 'script' is a console_script
-    entry_point = get_entry_point_from_console_script(script, self._distributions)
+    dist, entry_point = get_entry_point_from_console_script(script, self._distributions)
     if entry_point:
       self.set_entry_point(entry_point)
+      TRACER.log('Set entrypoint to console_script %r in %r' % (entry_point, dist))
       return
 
     # check if 'script' is an ordinary script
-    script_path, _, _ = get_script_from_distributions(script, self._distributions)
-    if script_path:
+    dist, _, _ = get_script_from_distributions(script, self._distributions)
```

Comment:
Do you know what the `_` are? If possible, I find `_foo, _bar` more readable while still expressing intent. 

Current is totally valid though, and is what we had before.

Reply:
Yeah, location and content (of the script). I'll leave as-is since its the file and even wider repo convention.